crates/stages/src/stages/hashing_storage.rs

```diff
@@ -172,12 +172,10 @@ impl<DB: Database> Stage<DB> for StorageHashingStage {
 
         // Aggregate all transition changesets and make list of accounts that have been changed.
         tx.cursor_read::<tables::StorageChangeSet>()?
-            .walk((from_transition_rev, H160::zero()).into())?
-            .take_while(|res| {
-                res.as_ref()
-                    .map(|(TransitionIdAddress((k, _)), _)| *k < to_transition_rev)
-                    .unwrap_or_default()
-            })
+            .walk_range(
+                (from_transition_rev, Address::zero()).into(),
+                (to_transition_rev, Address::zero()).into(),
```

Comment:
I'm not familiar with how walk range works,

what's the difference here? getting rid of the take while condition that is now performed in walk_range?

Reply:
This PR is just a refactor to avoid the repetitive patten

Reply:
> can't really comment on how useful this is. but makes sense to. wdyt @rakita
> 
> def needs a test

@mattsse what gakons said here, it is utility, it refactors some code